wurst/_handles/primitives/String.wurst

```diff
@@ -251,6 +253,58 @@ public function string.format(string par1, string par2, string par3, string par4
 				.replace("{8}", par9)
 				.replace("{9}", par10)
 				
+/** Replaces consecutive repeats of the same string with a single instance 
+	Turns "strstr" into "str" */
+public function string.reduce(string reduced) returns string
```

Comment:
What's the primary use case for this method? Feels like it belongs in a `StringExtensions` or `StringUtils` library, rather than the standard library.

Reply:
The primary use for it is when you're templating strings, and have missing values (intentionally), so you would turn "something  something    something" to "something something something", it's a niche use but it's pretty nice to have. I moved it to StringUtils anyways, you're right.